.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -284,7 +284,11 @@ jobs:
           echo "opts=$CMAKE_EXTRA_ARGS" >> $GITHUB_OUTPUT
         else
           if [ "${{ contains(inputs.target_devices, 'ext_oneapi_hip')  }}" == "true" ]; then
-            echo 'opts=-DHIP_PLATFORM="AMD" -DAMD_ARCH="gfx1031"' >> $GITHUB_OUTPUT
+            if [ "${{ runner.name }}" == "cp-amd-runner" ]; then
```

Comment:
I'm a bit hesitant about hard coding the runner name here. Just like we have `extra_lit_opts`, can we have another matrix input, something like `extra_lit_build_opts`, for passing runner specific build options? Or perhaps we can reuse `inputs.extra_cmake_args`?

Reply:
we could, but wouldn't we have to have the same hardcoded check at the place where we decide what to pass for `extra_lit_opts`? I think the real solution is running sycl-ls and grepping the output for the architecture, but I didn't want to overengineer it if the solution in this PR is acceptable.

Reply:
sure, i can own it actually, seems at least somewhat interesting.

https://github.com/intel/llvm/issues/16057